build.gradle

```diff
@@ -11,7 +11,7 @@
 
 buildscript {
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.1.0-SNAPSHOT")
```

Comment:
https://github.com/opensearch-project/security/pull/1948 is already in flight to update to 2.2 - which should be merged shortly.  Once that is done lets rebase to update to the new gradle version